recipe/meta.yaml

```
@@ -172,10 +176,10 @@ outputs:
         - async-timeout
         - colorful
         - grpcio
-        - memray  # [not win]
         - opencensus
         - prometheus_client >=0.7.1
-        - py-spy >=0.2.0
+        - py-spy >=0.2.0  # py<312
+        - py-spy >=0.4.0  # py>=312
```

Currently these are just comments, which means you'll get both constraints for all versions
```suggestion
        - py-spy >=0.2.0  # [py<312]
        - py-spy >=0.4.0  # [py>=312]
```